board/20160329-minutes.md

```diff
@@ -0,0 +1,217 @@
+# Ordinary General Assembly of RubyBelgium
+
+## Agenda
+
+https://slack-files.com/files-pri-safe/T02KPU3QY-F0WA1SL77/ordinarygeneralassembly-march2016.pdf?c=1459276026-33e89fe60970adb9a9f287e76f9c6355e546b8d0
+TODO Move this somewhere public, like github?
+
+**Present**: anthony `F`, boris `F`, cecile `F`, joren `F`, lien `F`, marius `F`, norydev `F`, oana `F`, olivier `F`, silvana `F`, simon `F`, stan `F`, tiago `F`, toch `F`, and tomasras `F`
+**Excused**: Edward `F`, Hannes `F`, Kevin `F`, Thomas Gautier `F`, and Yannick `F`
+**Late**: apneadiving `F`, bram `A`, mlainez `F`, nathan `A`, and yuri `F`
+
+`F` = Full Member
+`A` = Associate Member
+
+**Moderator**: Toch
+**Secretary**: Simon
+**Proxies**: Edward (to Toch), Hannes (to Toch), Kevin (to Stan), Thomas Gautier (to Brois), and Yannick (to Toch)
+
+## Welcome
+
+The moderator welcomes and thanks everyone present for the help and participation over the previous year. He is happy with the dynamics created and explains the rules to take the word to keep this online assembly as organized as possible.
```

Comment:
`to take the word` is not English I think.
I would use `to take the floor` or simpler `to speak`